### PR TITLE
Add build-service

### PR DESCRIPTION
**Pull request by Michkov**
Adding build-service supporting KCP

### CODE REVIEW

File: components/build-service/base/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+resources:
+- https://github.com/redhat-appstudio/build-service/config/kcp?ref=964b86001e08721e678fa21deb1142238894eed0
```
Comment (Michkov) [inline components/build-service/base/kustomization.yaml:2]: needs to be updated

File: components/build-service/base/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+images:
+- name: quay.io/redhat-appstudio/build-service
+  newName: quay.io/redhat-appstudio/pull-request-builds
+  newTag: buildserv-964b86001e08721e678fa21deb1142238894eed0
```
Comment (Michkov) [inline components/build-service/base/kustomization.yaml:10]: needs to be updated

**Review by mmorhun — APPROVED** [review]

File: components/build-service/overlays/kcp-unstable/apiexport.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      resource: "configmaps"
+    - group: ""
+      resource: "namespaces"
+    - group: ""
+      resource: "events"
+    - group: ""
+      resource: "serviceaccounts"
+    - group: "tekton.dev"
+      resource: "pipelineruns"
+      identityHash: pipeline-service
```
Comment (MatousJobanek) [inline components/build-service/overlays/kcp-unstable/apiexport.yaml:19]: shouldn't this contain the identity hash as well?

**Review by MatousJobanek — COMMENTED** [review]
Looks good - just one comment

File: components/build-service/overlays/kcp-unstable/apiexport.yaml (hunk 3 above)
Comment (Michkov) [reply components/build-service/overlays/kcp-unstable/apiexport.yaml:19]: Currently there is no deployment in kcp-unstable, but yeah comment would be great

**Pull request merged**